mobile/src/main/java/com/gelakinetic/mtgfam/FamiliarActivity.java

```diff
@@ -865,7 +865,11 @@ private boolean processIntent(@NonNull Intent intent) {
                                 throw new Exception("Not Found");
                             }
                         } else if ((queryParam = data.getQueryParameter("name")) != null) {
-                            cursor = CardDbAdapter.fetchCardByName(queryParam,
+                            String cardName = queryParam;
+                            if (queryParam.matches("\\+\\[.+\\]") { // See #458
```

Comment:
You're missing a close parenthesis on this line, but otherwise the changes work as expected. I tested against the 7 links you posted (https://github.com/AEFeinstein/mtg-familiar/issues/458#issuecomment-460068504). Please fix this before I accept the PR.

Reply:
Will do! Sorry for the silly mistake! And thanks :)